Formula/p/pivit.rb

```diff
@@ -0,0 +1,24 @@
+class Pivit < Formula
+  desc "Sign and verify data using hardware (Yubikey) backed x509 certificates (PIV)"
+  homepage "https://github.com/cashapp/pivit"
+  url "https://github.com/cashapp/pivit/archive/refs/tags/v0.6.0.tar.gz"
+  sha256 "67a2f353c820aad9c9a4cc937ba256c75cb36044baa0578b74a1bfbdf4e6bd90"
+  license "MIT"
+
+  depends_on "go" => :build
+
+  on_linux do
+    depends_on "pkg-config" => :build
+    depends_on "pcsc-lite"
+  end
+
+  def install
+    system "make", "pivit"
+    bin.install "pivit"
```

Comment:
Seeing the simplicity of the makefile, let's do this instead.
```suggestion
    system "go", "build", *std_go_args, "./cmd/pivit"
```

Reply:
Done! :)

Reply:
Doesn't it make more sence to use the Makefile provided with the tool, instead of letting homebrew be responsible for crafting the exact commands?

If the compilation command happens here, what would happen if this tool gets updated and the build process gets more complicated?